packages/kbn-ftr-common-functional-services/services/saml_auth/saml_auth_provider.ts

```diff
@@ -110,9 +116,12 @@ export function SamlAuthProvider({ getService }: FtrProviderContext) {
       const apiKeyHeader = { Authorization: 'ApiKey ' + apiKey.encoded };
 
       log.debug(`Created api key for role: [${role}]`);
-      return { apiKey, apiKeyHeader, cookieHeader: adminCookieHeader };
```

Comment:
to avoid misuse: this function returns only API key + header with role-scoped privileges. To get `Cookie` header for specific role, use the separate function